Mathlib/LinearAlgebra/Dimension.lean

```diff
@@ -728,6 +728,11 @@ theorem linearIndependent_le_span {ι : Type _} (v : ι → M) (i : LinearIndepe
   exact le_top
 #align linear_independent_le_span linearIndependent_le_span
 
+/-- A version of `linearIndependent_le_span` for `Finset`. -/
+theorem linearIndependent_le_span_finset {ι : Type _} (v : ι → M) (i : LinearIndependent R v)
+    (w : Finset M) (s : span R (w : Set M) = ⊤) : (#ι) ≤ Fintype.card w := by
+  simpa only [Finset.coe_sort_coe, Fintype.card_coe] using linearIndependent_le_span v i w s
+
```

Comment:
Please don't forward-port things until the mathport output is ready (https://leanprover-community.github.io/mathlib-port-status/file/linear_algebra/dimension shows that it is not). Porting things manually leads to errors, and you've made one here.

Reply:
Done.